src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Amounts.tsx

```diff
@@ -29,40 +30,44 @@ export default function Amounts() {
 
         return (
           <div
-            className="flex relative mb-6 border-b border-prim pr-2 pb-1 pt-6 items-center"
+            className={`relative grid grid-cols-[auto_1fr_auto] items-center gap-2 md:gap-4 w-full p-4 ${
+              errors?.amounts?.at && errors.amounts.at(i)?.value?.message
```

Comment:
is this style required in design? 
<img width="497" alt="image" src="https://user-images.githubusercontent.com/89639563/223010488-91b76caa-da15-4723-8ba0-d3125d6a85cc.png">

this field would be inconsistent to many fields throughout the app 
unless we all make our fields look like this when invalid

Reply:
> is this style required in design?

It's not required, but neither is the error message itself. 

> this field would be inconsistent to many fields throughout the app
unless we all make our fields look like this when invalid

I see your point, will revert for now.